package.json

```diff
@@ -27,6 +27,18 @@
     "eslint-config-godaddy": "^4.0.1"
   },
   "bin": {
-    "orglinter": "./src/bin/cli.js"
-  }
+    "orglinter": "./bin/cli.js"
+  },
+  "keywords": [
+    "github",
+    "linting",
+    "organization-management"
+  ],
+  "bugs": {
+    "url": "https://github.com/godaddy/orglinter/issues"
+  },
+  "files": [
+    "bin/",
+    "src/"
```

Comment:
If you are specifying the files for publishing, then you shouldn't need an `.npmignore`. <https://docs.npmjs.com/cli/v6/configuring-npm/package-json#files>

Reply:
`.npmignore` is a block-list
and `package.json`'s `files` property is an allow-list.

> You can also provide a .npmignore file in the root of your package or in subdirectories, which will keep files from being included. At the root of your package it will not override the "files" field, but in subdirectories it will.

So it could be useful if you combine your tests inside `src/` **if** you have a `src/.npmignore` and not the root one.  However, I'd recommend against having your tests colocated with your source.

Reply:
Oh interesting. Yeah I intend to have `test/` at the top level, alongside `src/` so that part wouldn't be an issue. I only ended up adding these both after finding them in https://github.com/godaddy/gasket/tree/master/packages/gasket-cli, so it kinda just seemed like, "This is the way."

Is there generally a preference for which way to go?

Reply:
Its really preference and what you want to optimize for.
`.npmignore` is good for blocking things if you have things mixed together or if you're frequently adding new files in new places that you want in your package
`files` is good for only allowing certain things and is good if you're not adding new file locations all the time and don't have mixed file types.

Reply:
![this is the way](https://media1.tenor.com/images/adbc199a37fac324e2b4bb26b5609fe5/tenor.gif)